ibc-core/ics24-host/types/src/path.rs

```diff
@@ -1219,42 +1219,42 @@ mod tests {
     #[case("ports/transfer", Path::Ports(PortPath(PortId::transfer())))]
     #[case(
         "channelEnds/ports/transfer/channels/channel-0",
-        Path::ChannelEnd(ChannelEndPath(PortId::transfer(), ChannelId::default()))
+        Path::ChannelEnd(ChannelEndPath(PortId::transfer(), ChannelId::new(0)))
```

Comment:
you may create a `zero()` method for these cases.

Reply:
I was kind of hesitant of creating new methods and as a result just used the existing one. Moving forward, I'll prioritize refactoring and implementing new methods when necessary. Thanks so much for the review.